sdk/keyvault/azure-security-keyvault-common/src/keyvault_pipeline.cpp

```diff
@@ -70,7 +70,8 @@ std::unique_ptr<Azure::Core::Http::RawResponse> _internal::KeyVaultPipeline::Sen
     case Azure::Core::Http::HttpStatusCode::NoContent:
       break;
     default:
-      throw KeyVaultException("Key Vault Keys error response received: ", std::move(response));
```

Comment:
It was not a mistake, but it was a workaround because at the time I was calling the Exception cttor, the raw-response was not yet parsed for the error code/msg.